src/mpc/structs.jl

```diff
@@ -42,6 +56,23 @@ Base.@kwdef struct MPCPV
 end
 
 
+"""
+    MPCWind
+```julia
+Base.@kwdef struct MPCWind
+    size_kw::Real = 0
+    production_factor_series::Union{Nothing, Array{Real,1}} = nothing
+end
+```
+"""
+Base.@kwdef struct MPCWind
+    size_kw::Real = 0
+    production_factor_series::Union{Nothing, Array{Real,1}} = nothing
+    om_cost_per_kw::Real = 0.0
+    operating_reserve_required_fraction::Real = 0.0
```

Comment:
I thought we weren't including this yet. Cause the op res constraints aren't being added in MPC right now.